libs/service-portal/information/src/components/PersonalInformation/Forms/ProfileForm/components/FormButton.tsx

```diff
@@ -27,11 +27,9 @@ export const FormButton: FC<React.PropsWithChildren<Props>> = ({
         {children}
       </Button>
     ) : (
-      <button type="submit" disabled={disabled}>
-        <Button variant="text" size="small" disabled={disabled}>
-          {children}
-        </Button>
-      </button>
+      <Button type="submit" variant="text" size="small" disabled={disabled}>
```

Comment:
I remember this code!
The text variant will render the button as span I believe. So you probably need to specifically set the render as button.
```suggestion
      <Button type="submit" as="button" variant="text" size="small" disabled={disabled}>
```